backends/rapidpro/urn.go

```diff
@@ -248,6 +259,8 @@ UPDATE
 SET 
 	channel_id = :channel_id, 
 	contact_id = :contact_id, 
+	identity = :identity, 
+	path = :path, 
```

Comment:
This change means we need to make sure all places that call `updateContactURN` are passing fully populated URN objects. Looking around it looks like `setDefaultURN` calls `contactURNsForContact` which doesn't select `path`. So need to update that.

Reply:
Was this addressed?

Reply:
I created a separated SQL ([R268](https://github.com/nyaruka/courier/pull/287/files#diff-01ee8537f8a93225f573ebcc5fd7ea4eR268)) for this update that's be used by a separeted function ([R299](https://github.com/nyaruka/courier/pull/287/files#diff-01ee8537f8a93225f573ebcc5fd7ea4eR299)).